README.md

```diff
@@ -49,6 +49,19 @@ $ tox
   congratulations :)
 ```
 
+## Citing Archspec
+
+If you are referencing Archspec in a publication, please cite the following
```

Comment:
`Archspec` -> `archspec`? (same in title above)

Reply:
fixed

Reply:
It still looks the same to me? 🤔

Reply:
oh I thought you meant the title and missed this.  Title is fixed.  This is also fixed now.